docs/section-7/sensors.md

```diff
@@ -28,28 +26,24 @@ Here is a list of encoder terminology:
 
 - Magnetic: uses a phenomenon known as the [Hall Effect](https://en.wikipedia.org/wiki/Hall_effect_sensor) to measure disturbances in a magnetic field.
 
-- EPR: edges-per-revolution; the total number of falling and rising edges per revolution.
+- EPR: edges-per-revolution - the total number of falling and rising edges per revolution.
 
 - Full period: four edges across the A and B channels.
 
 - PPR: pulses-per-revolution.
 
-- CPR: cycles-per-revolution; the same thing as the number of full periods per revolution.
+- CPR: cycles-per-revolution - the same thing as the number of full periods per revolution.
 
 ## Gyroscopes
-Gyroscopes measure the rotation of components on the robot. Usually, when working with a gyro, you will be using the nav-X gyro onboard the RoboRIO, shown below. In the bottom-left corner of the image, you can see the coordinate system used.
+Gyroscopes measure the rotation of components on the robot. Usually, when working with a gyro, you will be using the navX gyro onboard the RoboRIO, shown below. The coordinate system used by the navX is circled.
 
-![nav-X](https://andymark-weblinc.netdna-ssl.com/product_images/navx-mxp-robotics-navigation-sensor/5bd3589f61a10d293f9646ad/zoom.jpg?c=1540577439)
+![navX](navx.jpg)
```

Comment:
Not clearly free to reuse. Leave it out. Refer the reader to the product page and describe where the coordinate system is indicated.